### PR TITLE
improvement: bump alpine, node and gcloud SDK version in support

### PR DESCRIPTION
**Pull request by mkhq**
**What this PR does / why we need it**:

Update node, alpine and the gcloud SDK versions:

- node from 14.19.0 -> 14.19.3
- alpine from 3.14 -> 3.16
- GCloud SDK from 331.0.0 to 390.0.0. Note that there are some breaking changes detailed in the changelog: https://cloud.google.com/sdk/docs/release-notes

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: images/circleci-runner/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 FROM google/cloud-sdk:331.0.0 as gcloud
```
[the remark this answers is not in the document]
Comment (vvagaytsev) [reply images/circleci-runner/Dockerfile:2]: @edvald just to confirm, should we consider these image version updates as `improvement` or `chore` changes? Looks like all of these are mostly related to build steps in the ci pipeline, so I guess chore is fine here.

[the remark this answers is not in the document]
Comment (edvald) [reply images/circleci-runner/Dockerfile]: I think chore is fine here, unless there is something specific being addressed in the version updates that users should be aware of.

**Review by vvagaytsev — APPROVED** [review]

File: images/circleci-runner/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (vvagaytsev) [reply images/circleci-runner/Dockerfile]: Ok, it's ok to keep it as chore in circleci-runner. I'm not sure about `gcloud.Dockerfile`. How do we use that file? Is it exposed to the cloud customers who use GCP? Or is it used only by circleci?

[the remark this answers is not in the document]
Comment (mkhq) [reply images/circleci-runner/Dockerfile]: I've updated to improvement and detailed the bumped versions.

**Pull request merged**